src/ui/Help.jsx

```diff
@@ -12,24 +12,30 @@ const controlOptions = [
   { keys: ['C'], action: 'Toggle Camera' },
   { keys: ['R'], action: 'Reset' },
   { keys: ['E'], action: 'Editor' },
+  { keys: ['X'], action: 'Sfx' },
   { keys: ['I'], action: 'Help' },
 ]
 
 export function Help() {
   const set = useStore((state) => state.set)
   const open = useStore((state) => state.help)
+  const sfx = useStore((state) => state.controls.sfx)
+
   return (
-    <div className="controls">
-      {!open && <button onClick={() => set({ help: true })}>i</button>}
-      <div className={`popup ${open ? 'open' : ''}`}>
-        <button className="popup-close" onClick={() => set({ help: false })}>
-          i
-        </button>
-        <div className="popup-content">
-          <Keys />
+    <>
+      <div className={`${sfx ? 'sound' : 'nosound'}`}></div>
```

Comment:
maybe extract this to its own ui component

Reply:
same as above, why overhead if just about a class change?

Reply:
flipping a class seems so minor, i think it's fine. let's get these things out wholesale with styled-components or something like that later on. that makes the process quicker. and once we have a better css setup then this stuff will clear itself. 🙂